tools/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ if(NOT Python_EXECUTABLE)
     return()
 endif()
 
-find_program(CLANG_FORMAT clang-format PATHS /opt/rocm/llvm $ENV{HIP_PATH})
+find_program(CLANG_FORMAT clang-format PATHS /opt/rocm/llvm/bin /opt/rocm*/llvm/bin $ENV{HIP_PATH})
```

Comment:
Please use `PATH_SUFFIXES` instead because the above change does not consider Windows
```cmake
find_program(CLANG_FORMAT clang-format PATHS /opt/rocm/llvm ENV HIP_PATH PATH_SUFFIXES bin)
```
Also, the `PATHS` argument does not recognize and support wildcards, so `/opt/rocm*/llvm` will not work as intended.